scripts/weather_providers/open_meteo.js

```diff
@@ -61,7 +62,7 @@ function getWeatherDescription(weatherCode) {
 // Map weather codes to weather icons
 function getWeatherIcon(weatherCode) {
   const icons = {
-    0:  "clear_sky",
+    0:  "clear_sky_day",  // TODO: Add logic to determine day or night
```

Comment:
There was [previously logic for day or night](https://github.com/sashakarcz/lofiweather/blob/9849813f0718b252e15de6244b683afa79112043/script.js#L58)

Reply:
Resolving and pulling commit.